ui/pages/login/index.js

```diff
@@ -97,9 +103,11 @@ export default function Login() {
       const data = await res.json()
 
       if (data.passwordUpdateRequired) {
+        const query = next ? { user: data.userID, next } : { user: data.userID }
```

Comment:
nit: optionally you can just put this declaration in the object below, e.g.

`router.replace({query: ...})`